go/vt/vtgate/executor.go

```diff
@@ -59,7 +59,7 @@ var (
 	errNoKeyspace     = vterrors.Errorf(vtrpcpb.Code_FAILED_PRECONDITION, "no keyspace in database name specified. Supported database name format (items in <> are optional): keyspace<:shard><@type> or keyspace<[range]><@type>")
 	defaultTabletType topodatapb.TabletType
 
-	queriesProcessed = stats.NewCountersWithSingleLabel("QueriesProcessed", "Queries processed at vtgate by plan type", "Plan")
+	queriesProcessed = stats.NewCountersWithMultiLabels("QueriesProcessed", "Queries processed at vtgate by plan type, keyspace and table", []string{"Plan", "Keyspace", "Table"})
```

Comment:
Adding labels to an existing metric can confuse downstream systems, including Prometheus. This isn't necessarily the end of the world, but overall I think it would be cleaner to use a new metric name if we're going to change the labels.

Maybe we rename this to simply "Queries"?

I also think we should extend `QueriesRouted` to include the same set of labels since it's nice to keep those two in sync to measure the percentage of scatter queries that are happening.

I'd then rename that one to "ShardQueries"